client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -230,6 +300,7 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
                                     size='small'
                                     className={classes.otherTextField}
                                     placeholder='הזן סימפטום...'
+                                    value={otherSymptom}
```

Comment:
You can't type anything onBlur when you set value. Check it locally.

Reply:
then why use onBlur?

Reply:
Because you don't want the event value of each typing letter, only when the typing has come to an end.